File: programmers/default_findDecimal.js
/*
	rank:	1
	title:	소수 찾기
	link: 	https://programmers.co.kr/learn/courses/30/lessons/12921
*/

function solution(n) {
    let arr = new Array(n - 1).fill(true);

    for (let i = 2; i <= arr.length; i++) {
        for (let j = i * i; j <= n; j += i) {
            arr[j - 2] = false;
        }
    }

    return arr.filter(value => value).length;
}

File: programmers/default_findDecimal.py
# rank:	    1
# title:	소수 찾기
# link: 	https://programmers.co.kr/learn/courses/30/lessons/12921

def solution(n):
    arr = [True for value in range(2, n + 1)]

    for i, value in enumerate(arr):
        for j in range((i + 2) ** 2, n + 1, i + 2):
            arr[j - 2] = False

    return len([value for value in arr if value])